input/fsh/SD_Extensions.fsh

```diff
@@ -333,3 +333,32 @@ Expression: "extension.where(url = 'energy').exists() implies \n
                 )\n
               )"
 XPath: "true()"
+
+Invariant:   TG263BED
+Description: "Unit must match TG263 BED Regexp"
+Expression:  "$this.toString().matches('(CGE$)|(BED|EQD2Gy)([(]a\\/b=[0-9]*.?[0-9]*[)])?$')"
+Severity:    #warning
+
+Extension: BiologicallyEffectiveDose
+Id: codexrt-radiotherapy-biologically-effective-dose
+Title: "Biologically Effective Dose Extension"
```

Comment:
The other extension titles have no Extension at the end (almost, we already have one exception Radiotherapy Volume Extension). Also remove here?